packages/signer/keyring-eth/src/internal/app-binder/command/SignTransactionCommand.ts

```diff
@@ -45,14 +55,36 @@ export class SignTransactionCommand
     this.args = args;
   }
 
+  private getP1(): number {
+    const { flow, isFirstChunk } = this.args;
+    if (flow === SignTransactionFlow.LEGACY) {
+      return isFirstChunk ? 0x00 : 0x80;
+    }
+
+    return isFirstChunk ? 0x01 : 0x00;
+  }
+
+  private getP2(): number {
+    const { flow } = this.args;
+    if (flow === SignTransactionFlow.LEGACY) {
+      return 0x00;
+    }
+
+    if (flow === SignTransactionFlow.STORE_AND_START) {
+      return 0x01;
+    }
+
+    return 0x03;
```

Comment:
1. If the ["store only"(0x02)](https://github.com/LedgerHQ/generic_parser/blob/master/specs.md#sign) does not need to be handled here, it's OK;
2. Enum default value can be used here:
```suggestion
return flow === SignTransactionFlow.START ? 0x03 : flow;
```

Reply:
not sure if it's better but it could also be a const mapping such as:

    const FLOW_TO_P2: Record<SignTransactionFlow, number> = {
      [SignTransactionFlow.LEGACY]: 0x00,
      [SignTransactionFlow.STORE_AND_START]: 0x01,
      [SignTransactionFlow.START]: 0x03,
    };

Reply:
@jiyuzhuang the store is handle in another command StoreTransaction it "cannot" be handle here.

@paoun-ledger if we need more "logic", the function can handle it better than just the object